src/library_stack_trace.js

```diff
@@ -1,31 +1,47 @@
 var LibraryStackTrace = {
 
-#if ASSERTIONS && MINIMAL_RUNTIME
-  $demangle__deps: ['$warnOnce'],
+  _demangle_failed: 0,
+
+  $demangle__deps: [
+#if MINIMAL_RUNTIME
+  '$stackSave', '$stackAlloc', '$stackRestore'
+#if ASSERTIONS
+    , '$warnOnce'
+#endif
+#endif
+#if DEMANGLE_SUPPORT
+#if MINIMAL_RUNTIME
+    ,
```

Comment:
does this `,` need an ifdef all for itself? We can add a `,` on the `_demangle_failed` line, like we do for `warnOnce` I think?

Reply:
No, the conditions were separate/orthogonal. However removed by using `demangle.failed`.